Bio/ExPASy/Enzyme.py

```diff
@@ -55,7 +55,7 @@ def read(handle):
 
 
 class Record(dict):
-    """\
+    r"""\
```

Comment:
If this does seem to be a false positive from ``pydocstyle``, we should make a simple test case and report a bug to them. But maybe I'm just overlooking a slash elsewhere in this docstring while reading this in GitHub?